projects/kit/components/input-number/input-number.options.ts

```diff
@@ -0,0 +1,31 @@
+import type {Provider} from '@angular/core';
```

Comment:
Let's use this util to create token and helper:
https://github.com/taiga-family/taiga-ui/blob/main/projects/cdk/utils/di/create-options.ts

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/10031/commits/21703a58cae39021aef32a9ecf4cb297188a55ec